NewUserGuide.md

```diff
@@ -1,3 +1,11 @@
+---
+2	layout: page
```

Comment:
Ok I don't know why it was auto adding numbers it hadn't done that for me before :/